samples/bluetooth/opendroneid/src/main.c

```diff
@@ -0,0 +1,279 @@
+/* SPDX-License-Identifier: Apache-2.0
+ * Copyright 2024 NXP
+ *
+ * Referred from https://github.com/opendroneid/transmitter-linux
```

Comment:
You're missing the copyright entry for the original author.

Reply:
fixed in https://github.com/zephyrproject-rtos/zephyr/pull/82034